command/helper/config.go

```diff
@@ -219,6 +232,11 @@ func (c *Config) mergeConfigWith(otherConfig *Config) error {
 		c.GRPCAddr = otherConfig.GRPCAddr
 	}
 
+	if otherConfig.Metrics.PrometheusAddr != "" {
+
```

Comment:
Nitpick warning: please remove the empty space

Reply:
Removed it